packages/components/src/Select/docs/labeling.tsx

```diff
@@ -1,11 +1,13 @@
-import { Select } from "@hopper-ui/components";
+import { Select, SelectField, SelectOption } from "@hopper-ui/components";
 
 export default function Example() {
     return (
-        <Select aria-label="Roles">
-            <Select.Option id="designer">Designer</Select.Option>
-            <Select.Option id="developer">Developer</Select.Option>
-            <Select.Option id="manager">Manager</Select.Option>
-        </Select>
+        <SelectField aria-label="Roles">
```

Comment:
[Question] This is mandatory? So a select cannot really live without a SelectField?

Reply:
Correct, a SelectField is required for a select.